.travis.yml

```diff
@@ -15,29 +15,30 @@ os:
 
 env:
   global:
+    - MAKEFLAGS="-j4"
     # For Coverity
     - secure: "MeTS1Pqa5gzx1nn/peW/9a5kq84bba3XYUljOfkCUqzuyGiERk/nmok+RW7skrgzboBlKxnNG8+ykKqHMwK9s9M83ezFxvEWXBcKEpmEQKkqXPI5hpMs6jGLTgpeuheSIzqHA3danV8iircp1GOiTLWA0pt/AOsNLZiaYBh0OiE="
   matrix:
-    - BUILD_EXTRA_FLAGS="--without-x"
-    - TRACE_CODE_COVERAGE=true
-    - OSX_ONLY=true
+    - MODE=compilation
+    - MODE=distcheck
+    - MODE=coverage
 matrix:
+  include:
+    - os: linux
+      env: MODE=compilation BUILD_EXTRA_FLAGS="--without-x"
   exclude:
     - os: osx
-      env: BUILD_EXTRA_FLAGS="--without-x"
+      env: MODE=distcheck
     - os: osx
-      env: TRACE_CODE_COVERAGE=true
-    - os: linux
-      env: OSX_ONLY=true
+      env: MODE=coverage
 
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
       - deadsnakes
-    packages: [autoconf, automake, autotools-dev, bzip2, gcc-7, libtool,
```

Comment:
This was like so, to act as a guide as to what packages are required to build FontForge.

Reply:
Sounds reasonable and costs only a couple of seconds. Then please exclude commit d096e0e0ad627af730a7869aa204a7074a501801 during the merge.